db/migrate/20241105143353_regenerate_taxonomy_search_for_filters_with_override_false.rb

```diff
@@ -0,0 +1,10 @@
+class RegenerateTaxonomySearchForFiltersWithOverrideFalse < ActiveRecord::Migration[6.1]
+  def change
+    filters = Filter.where(role_id: Role.where(origin: nil).or(Role.where(builtin: 2))).where(override: false).where(taxonomy_search: nil)
```

Comment:
```ruby
    where(taxonomy_search: nil)
```
Was it in the previous iteration of commit? I thought we wanted to explicitly override any overridden searches, this will only add instead thus leaving overridden searches, which seems wrong...